src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -26,6 +27,8 @@ public class OtlpExporterOptions
     internal const string HeadersEnvVarName = "OTEL_EXPORTER_OTLP_HEADERS";
     internal const string TimeoutEnvVarName = "OTEL_EXPORTER_OTLP_TIMEOUT";
     internal const string ProtocolEnvVarName = "OTEL_EXPORTER_OTLP_PROTOCOL";
+    internal const string RetryFeatureEnvVarName = "OTEL_DOTNET_EXPERIMENTAL_OTLP_ENABLE_RETRY";
+    internal const string StorageEnVarName = "OTEL_DOTNET_EXPERIMENTAL_OTLP_PERSISTENT_STORAGE_DIR";
```

Comment:
Environment variable name is misleading as it does both persistent storage / retry.